### ISSUE TITLE
data as array

### ISSUE THREAD

**Issue by QQizi** [gitim/react-native-sortable-list#80]
I'm using an array instead of an object, works fine but had an issue poping because of the PropTypes.

Changed SortableList.js with : 

```
data: PropTypes.oneOfType([
      PropTypes.object.isRequired,
      PropTypes.array.isRequired
    ])
```

Works fine ;)

**Comment by EriveltonElias** [comment]
Hi @QQizi ! 

I add a pull request that resolves it. :)

#82

**Comment by gitim** [comment]
released fix in 0.0.20
